src/context-selection/section-filter-selector-bar-item/section-filter-selector-bar-item.jsx

```diff
@@ -67,7 +71,7 @@ export default function SectionFilterSelectorBarItem() {
             >
                 <MenuSelect
                     values={selectableOptions}
-                    selected={sectionFilter}
+                    selected={sectionFilter ?? null}
```

Comment:
I've kept the undefined/null distinction for the useEffect dependency, but I'm mapping undefined to null here in order for the default "All sections" to show up as highlighted when nothing is selected.